pkg/graph/schema/types/mto_milestone_note.graphql

```diff
@@ -45,7 +38,5 @@ extend type Mutation {
     input: MTOMilestoneNoteUpdateInput!
   ): MTOMilestoneNote! @hasRole(role: MINT_USER)
 
-  deleteMTOMilestoneNote(
-    input: MTOMilestoneNoteDeleteInput!
-  ): MTOMilestoneNote! @hasRole(role: MINT_USER)
+  deleteMTOMilestoneNote(id: UUID!): Boolean! @hasRole(role: MINT_USER)
```

Comment:
Hmm, any reason that this switched to a Boolean instead of the object? Both are in the code base, but I think returning the deleted object is a slightly preferable paradigm. (eg you can infer errors, but this is the last time to get a reference to the object). 

Thoughts?